src/components/form/index.jsx

```diff
@@ -33,8 +34,18 @@ const countriesWithStates = [UNITED_STATES, CANADA];
 
 const FormHeader = () => (
   <Form.Row className="flex-column mb-4 pl-1">
-    <h3 className="form-title">Verify your identity</h3>
-    <p className="form-subtitle">For security reasons, we need the following information to verify your identity.</p>
+    <h3 className="form-title mb-4">Verify your identity</h3>
+    <span className="d-inline-flex align-items-center form-subtitle">
+      <Icon src={WarningFilled} className="align-middle mr-2 text-danger" />
+      <strong>
+        IMPORTANT: You must enter your first/given and last/surname/family name exactly
+        as it appears on the identification (ID) you will present at the test center.
+      </strong>
+    </span>
+    <span className="form-subtitle">
+      If there is not an exact match, you will not be able to take
+      your test and you will not be reimbursed for any fees paid.
+    </span>
```

Comment:
With this changes this will look like similar like the image attached to the ticket

```suggestion
    <h3 className="form-title mb-4">Verify your identity</h3>
    <p className="form-subtitle">
      <Icon src={WarningFilled} className="align-middle mr-1 text-danger icon-header" />
      <strong className="pl-4">
        IMPORTANT: You must enter your first/given and last/surname/family name exactly as it appears
        on the identification (ID) you will present at the test center.
      </strong>{' '}
      If there is not an exact match, you will not be able to take your test and
      you will not be reimbursed for any fees paid.
    </p>
```

You must add this into `src/components/form/index.scss` in order to make it fully work

```css
.icon-header {
  position: absolute;
  top: 20px;
  right: 3px;
}

.pgn__icon.icon-header {
  height: 20px;
}
```


<img width="1363" height="868" alt="Image" src="https://github.com/user-attachments/assets/91fbe8a9-5770-400d-bda0-88794db73756" />

Reply:
@01001110J Why is the `{' '}` required after the `</strong>` tag?